browser/react/components/App.js

```diff
@@ -1,16 +1,17 @@
 import React from 'react';
-
 import '../../aframeComponents/scene-load';
 import '../../aframeComponents/aframe-minecraft';
+import AssetLoader from './AssetLoader';
 
 export default function App (props) {
   return (
+    // AssetLoader is a stateless component containing the a-assets for all of the React components
+    //   rendered via props.children. It must reside here because A-Frame requires a-assets to a
+    //   direct child of a-scene.
     <div style={{ width: '100%', height: '100%' }}>
       <a-scene id="scene" scene-load>
-        <a-entity geometry="primitive: sphere; radius: 1.25" position="0 1.25 -1" material="color: #EF2D5E"></a-entity>
-        <a-entity geometry="primitive: box; width: 1; height: 1; depth: 1" position="-1 0.5 1" material="color: #4CC3D9"></a-entity>
-        <a-entity geometry="primitive: cylinder; radius: 0.5; height: 1.5" position="1 0.75 1" material="color: #FFC65D"></a-entity>
-        <a-entity geometry="primitive: plane; width:4; height: 4" rotation="-90 0 0" material="color: #7BC8A4"></a-entity>
+        <AssetLoader />
```

Comment:
correct me if i'm wrong, but could there be a potential race condition here? could the props.children render before the assets are loaded on the scene?